README.md

```diff
@@ -42,39 +42,58 @@ For using it, it becomes:
 |Vagrant| http://localhost:8888 |
 | Vagrant, applying the advice above| http://appserver, with the possibility of going to http://portainer.appserver or http://traefik.appserver |
 
+### Subdomains registered on traefik
+read "_subdomain_.appserver"; e.g. you put in your browser traefik.appserver
+
+|Subdomain|What|Container|
+|---|---|---|
+|traefik|Traefik's statuspage (under login)|traefik|
+|kibana|Central logging (under login)|kibana|
+|portainer | Easier container mgmt|portainer|
+|status|MyAEGEE's statuspage|cachet|
+|my|MyAEGEE|oms-frontend|
+|www|Website|wordpress|
 
 [For more detailed usage guides see this usage tips page.](https://oms-project.atlassian.net/wiki/spaces/GENERAL/pages/23655986/Usage+tips)
 For container-specific usage guides see the container's repository.
 
+## Easy script
+There is a file called Makefile that gives some easy shortcut to do stuff. Invoke it in the following way:
+
+| Command | What |
+|---|---|
+| make init | (Run only the first time by vagrant) | 
+| make build | Build the containers registered in the .env file | 
+| make start | Runs the containers registered in the .env file | 
+| make bootstrap | (init, build, start) in this order | 
+| make monitor | If you didn't install kibana, then you may want to have a look at the logs through this | 
+| make live-refresh | Updates the containers to the new version (if any) and restarts them | 
+| make stop/restart/hard-restart | Just don't use them on the server, EVER | 
+| make bump | Only for development, updates the submodules |
+
+For now, if one wants to follow some specific logs, they have to invoke helper.sh manually e.g. 
+```
+./helper.sh --monitor container1 container2...containerN
+```
+
+Likewise, for now if one wants to execute a command on a container they have to invoke helper.sh manually e.g. 
+```
+./helper.sh --execute containername command
+```
+
+On first run of vagrant, the bootstrap target will be invoked
+
 ## Contribute
 [You can read more about contributing on our confluence.](https://oms-project.atlassian.net/wiki/spaces/GENERAL/overview)
 
 ## Issue tracker
-[We use JIRA as our preferred issue tracker.](https://oms-project.atlassian.net/projects/GENERAL/issues)
+[We use JIRA as our preferred issue tracker.](https://oms-project.atlassian.net/projects/MEMB/issues)
 
 ## Licence
 Apache License 2.0, see LICENSE.txt for more information.
 
-## Ports registered on traefik
-| Port | What | Container |
-|---|---|---|
-| 80,443 | standard entrypoint | traefik |
-| 5432 | not registered | postgres |
 
 ## Deployment script
 The types of deployment:
```

Comment:
can you add extra \n after this line? so Markdown would understand it's a list and display it properly

Reply:
I don't understand, this part has been removed

Reply:
no, the line after The types of deployment